apps/console/src/features/core/components/authenticator-accordion.tsx

```diff
@@ -89,6 +94,15 @@ export interface AuthenticatorAccordionItemInterface {
      * Title of the authenticator.
      */
     title: string;
+    /**
+     * Additional Options for the title.
+     */
+    titleOptions?: {
+        /**
+         * Flag to determine if the accordion title should be flex.
+         */
+        flex?: boolean;
```

Comment:
```suggestion
    /**
     * Flag to determine if the accordion title should be flex.
     */
    flex?: boolean;
```

Reply:
`flex` should be inside the `titleOptions` object. I believe the original indentation was accurate?